pkg/agent/flowexporter/types.go

```diff
@@ -51,19 +52,28 @@ type Connection struct {
 	Labels, LabelsMask []byte
 	// TODO: Have a separate field for protocol. No need to keep it in Tuple.
 	TupleOrig, TupleReply          Tuple
+	FlowKey                        Tuple
 	OriginalPackets, OriginalBytes uint64
 	ReversePackets, ReverseBytes   uint64
 	// Fields specific to Antrea
-	SourcePodNamespace            string
-	SourcePodName                 string
-	DestinationPodNamespace       string
-	DestinationPodName            string
-	DestinationServicePortName    string
-	IngressNetworkPolicyName      string
-	IngressNetworkPolicyNamespace string
-	EgressNetworkPolicyName       string
-	EgressNetworkPolicyNamespace  string
-	TCPState                      string
+	SourcePodNamespace             string
+	SourcePodName                  string
+	DestinationPodNamespace        string
+	DestinationPodName             string
+	DestinationServicePortName     string
+	IngressNetworkPolicyName       string
+	IngressNetworkPolicyNamespace  string
+	IngressNetworkPolicyRuleAction uint8
+	EgressNetworkPolicyName        string
+	EgressNetworkPolicyNamespace   string
+	EgressNetworkPolicyRuleAction  uint8
+	TCPState                       string
+	// fields specific to deny connections
+	// DeltaBytes and DeltaPackets are octetDeltaCount and packetDeltaCount over each active
+	// flow timeout duration.
+	DeltaBytes, DeltaPackets uint64
+	TotalBytes, TotalPackets uint64
```

Comment:
I understand the difference between `DeltaBytes` and `TotalBytes`. But I still don't understand the difference between `OriginalBytes` and `TotalBytes`.

Reply:
you are right.. `TotalBytes` is the same as `OriginalBytes`. Have it removed